modules/fastq/templates/merge_cram.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -euo pipefail
+
+main() {
+    IFS=' ' read -a cram_array <<< "!{crams}";
+    if [ ${#cram_array} -gt 1 ]
+    then
+        ${CMD_SAMTOOLS} merge -@ "!{task.cpus}" -o !{cramOut} --write-index "${cram_array}"
+    else
+        mv !{crams} "!{cramOut}"
+        exit 2
+    fi
+
+    if [ "!{isPairEnded}" == "true"]
+    then
+        ${CMD_SAMTOOLS} fixmate -u -m - - | \
```

Comment:
does this work? fixmate is reading from stdin, but no data is piped into the process.